src/ast/nodes.hpp

```diff
@@ -62,7 +70,24 @@ struct Function
   CodeBlock body;
 };
 
-using TranslationUnit = std::vector<Function>;
+using Method = Function;
```

Comment:
Not sure for this one. I would prefer not using an alias but it would also require adding some lines to call the function parsing under the method parsing.